src/App.jsx

```diff
@@ -1,5 +1,130 @@
+import { useEffect, useState } from 'react';
+import { Global, css } from '@emotion/react';
+import styled from '@emotion/styled';
+import InputForm from './components/InputForm.jsx';
+import TodoList from './components/TodoList.jsx';
+import Header from './components/Header.jsx';
+import bgImage from './grid.jpg';
+
+const GlobalStyles = () => (
+  <Global
+    styles={css`
+      @font-face {
+        font-family: 'RixXladywatermelonR';
+        src: url('https://fastly.jsdelivr.net/gh/projectnoonnu/2408-4@1.0/RixXladywatermelonR.woff2')
+          format('woff2');
+        font-weight: normal;
+        font-style: normal;
+      }
+
+      html {
+        margin: 0;
+        display: flex;
+        justify-content: center;
+        align-items: center;
+        height: 100vh;
+      }
+
+      body {
+        font-family: 'RixXladywatermelonR', sans-serif;
+        max-width: 500px;
+        padding: 10px;
+        text-align: center;
+        display: flex;
+        justify-content: center;
+        align-items: center;
+        scrollbar-width: thin;
+        margin: 0;
+      }
+
+      button {
+        font-family: 'RixXladywatermelonR';
+        display: flex;
+        justify-content: center;
+        align-content: center;
+        flex-wrap: wrap-reverse;
+      }
+    `}
+  />
+);
+
+const ContainerStyle = styled.div`
+  position: relative;
+  display: flex;
+  width: 450px;
+  height: 680px;
+  flex-direction: column;
+  background-color: #fef79fbd;
+  overflow: hidden;
+  ::after {
+    content: '';
+    position: absolute;
+    top: 0;
+    left: 0;
+    width: 100%;
+    height: 100%;
+    background-image: url(${bgImage});
+    background-size: cover;
+    background-blend-mode: darken;
+    opacity: 0.5;
+    z-index: -1;
+  }
+`;
+
 function App() {
-  return <div>React Todo</div>;
+  const userName = '옹헤';
```

Comment:
상수화 하여 사용하신 점 아주 좋습니다!!
네이밍 컨벤션 측면이긴 하지만 상수값을 사용할 때에는 `UPPER_SNAKE_CASE`를 사용한답니다! ([참고](https://ui.toast.com/fe-guide/ko_CODING-CONVENTION#%EC%83%81%EC%88%98%EB%8A%94-%EC%98%81%EB%AC%B8-%EB%8C%80%EB%AC%B8%EC%9E%90-%EC%8A%A4%EB%84%A4%EC%9D%B4%ED%81%AC-%ED%91%9C%EA%B8%B0%EB%B2%95snake-case%EB%A5%BC-%EC%82%AC%EC%9A%A9))

Reply:
또한 컴포넌트는 함수이기 때문에 상태가 업데이트 될 때마다 컴포넌트 내부에 선언된 변수는 매번 메모리에 재할당 됩니다! 
하지만 상수는 재할당 될 필요가 없기 때문에 주로 컴포넌트 밖에서 선언해 줍니다 🙂

```suggestion
const USER_NAME = '옹헤';

function App() {
  // 생략
}
```

[참고자료](https://duklook.tistory.com/566)